data/cache/impl.py

```diff
@@ -244,19 +244,32 @@ class RedisDataModelCache(DataModelCache):
     """
 
     def __init__(
-        self,
-        cache_config,
-        host="127.0.0.1",
-        port=6379,
-        password=None,
-        db=0,
-        ca_cert=None,
-        ssl=False,
+            self,
+            cache_config,
+            read_host="127.0.0.1",
+            read_port=6379,
```

Comment:
These new values should also be added to the default config.

Reply:
The `config.py` has the Memcache settings. Can we add this without causing any issues?

Reply:
i've added these settings in the enhancement repo https://github.com/quay/enhancements/pull/8

Reply:
If you add/change config values, defaults should be set/updated in `config.py`.

Reply:
See https://github.com/quay/quay/blob/master/config.py

Reply:
For example, with some of the host key name changes, the values in `config.py` are no longer valid.

Reply:
Nevermind, the default uses memcached